### ISSUE TITLE
Accessibility fix - support tabbing to question text and choice text

### ISSUE THREAD

**Issue by johneosborne** [KazuAlex/openpollslack#28]
For accessibility for people with screen readers, it should be possible to navigate the UI with a keyboard. When I use the tab key to navigate through a poll, I'm currently taken to the vote buttons without going to the question or choice text. To fix this, the tab key should take me first to the question text, then the choice text, then the button for that choice.
